doc/howto/network_acls.md

```diff
@@ -27,7 +27,7 @@ See {ref}`network-acls-groups` for more information.
 
 To list all ACLs, run:
 
-```
+```bash
```

Comment:
When we talk about each logical change being a commit, these changes would be a good example of a separate commit. Something like "Updated code blocks in ACLs section to use bash highlighting" along with an explanation as to why.

Then the actual content changes can be a separate commit explaining what has been changed.

Otherwise we are ending up with large commits that change a variety of different things in the docs.

Reply:
I would also encourage you to split up formatting changes from content changes, the former needing less focused review than the latter. If the two are mixed in together it is harder to know which bits to focus on.